test/async-hooks/async-hooks.status

```diff
@@ -9,6 +9,8 @@ prefix async-hooks
 [$system==win32]
 
 [$system==linux]
+# https://github.com/nodejs/node/issues/54809
```

Comment:
Tracking issue opened 10 minutes ago and still maked as flaky.

Reply:
The flakyness of the CI is extremely high that it's started to bug other contributors including me. We can wait for someone to pick up this flaky test and fix it, or we can mark it and improve the productivity of this whole project.

Reply:
Not sure what you mean by this? The test has been consistently flaky, I think @anonrig is just making sure there are tracking issues for each of these?

Reply:
I mean no one has looked into these failures (some of them are infra related), now tracking issues are being created and the corresponding tests immediately marked as flaky. It does not seem right to me.

Reply:
ok? tracking issues are literally how we keep track of these things in order to bring them to folks attention so they can be looked at ... so folks *know* to look at them. In the meantime we're trying to clear the current log jam of PRs that can't land because of a growing number of flaky tests. We all agree they need to be looked at but we have to start somewhere.

Reply:
I already had that same discussion with @anonrig on a similar PR he made months ago. Once a test is marked flaky it stays flaky because it is no longer a problem. It is no longer a PITA. If we don't have the time to resolve a PITA, we will never have the time to remove the flaky designation from these tests.

Reply:
We gotta select our battles.

I think not having time to remove the flaky designation is better than having frustrated collaborators that eventually churn.

Reply:
I disagree. We should should fix existing issues (instead of hiding them) before adding new features. With this approach we will have an ever growing number of flaky tests.

Reply:
Ok, then let's fix them *AND* mark them flaky while they are being fixed so that we're not indefinitely blocking other work. If folks don't want an ever growing list of flaky tests then folks need to actively help fix them. Personally I've already started working on investigating some of these.